service/csv/pokemon.go

```diff
@@ -0,0 +1,225 @@
+package service_csv
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/rayl17/academy-go-q42021/model"
+
+	parameters "github.com/rayl17/academy-go-q42021/global"
+)
+
+type ServiceInterface interface {
+	GetPokemonByName(name string) (*model.Pokemon, error)
+	GetPokemonByID(id string) (*model.Pokemon, error)
+	getPokemons() []model.Pokemon
+	SearchPokemonApi(name string) (string, error)
+	GetPokemonsByTypes(types string, items int, items_per_worker int) (*[]model.Pokemon, error)
+}
+
+type PokemonService struct {
+	servicePath string
+}
+
+func NewPokemonService(path string) (*PokemonService, error) {
+
+	return &PokemonService{
+		servicePath: path,
+	}, nil
+}
+
+// this method recives a name and return the pokemon with that name
+func (ps *PokemonService) GetPokemonByName(name string) (*model.Pokemon, error) {
+
+	for _, pokemon := range ps.getPokemons() {
+		if pokemon.Name == name {
+			return &pokemon, nil
+		}
+	}
+
+	return nil, errors.New("Pokemon not found ")
+}
+
+/// This method recives a id to look for and returns the pokemon with that id ,
```

Comment:
Be consistent; only two `/` per comment. Fix this here and all other instances.

Remember, the comment should start with the name of the element you are documenting.
```suggestion
// GetPokemonByID receives an ID to look for and returns the pokemon with that ID.
```